static/js/compose_actions.js

```diff
@@ -494,6 +494,25 @@ export function on_topic_narrow() {
     $("#compose-textarea").trigger("focus").trigger("select");
 }
 
+export function get_quoted_message(message) {
+    // Final message looks like:
+    //     @_**Iago|5** [said](link to message):
+    //     ```quote
+    //     message content
+    //     ```
+    let content = $t(
+        {defaultMessage: "{username} [said]({link_to_message}):"},
+        {
+            username: `@_**${message.sender_full_name}|${message.sender_id}**`,
+            link_to_message: `${hash_util.by_conversation_and_time_url(message)}`,
+        },
+    );
+    content += "\n";
+    const fence = fenced_code.get_unused_fence(message.raw_content);
+    content += `${fence}quote\n${message.raw_content}\n${fence}`;
+    return content;
+}
```

Comment:
Is this function unchanged from the original? If so, we can make this function extraction a prep commit and just merge it.